### DISCUSSION TITLE
Git Tags to news feed

### DISCUSSION THREAD

**Discussion by Thorium** [community/community#127533] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I like the news feed, but I cannot replace my Twitter feed browsing with that, because my friends are not productive enough, or the feed is not displaying enough news to me.

So I'd like to have options in the current news feed to display:

 a) **git tags** added to repositories. Many times people have already spend their time to gather the releasenotes, and are too lazy to create separate release documentation in Github, but instead add just a tag to repository.

b) Maybe even option to list direct **commits to main/master** from the people I follow, not only PRs.

![image](https://github.com/community/community/assets/229355/8dc932e4-7d9d-46bf-b893-0e2b641b44f9)

**Comment by LiteBrite82** [comment]
Hi @Thorium !

Thanks for the feedback!!

This community is managed by GitHub employees, but it is not a support forum and there is no SLA or guarantee each post will get a response from a community manager or product leader.

If your discussion provided product feedback or a feature request we encourage you to review the "Making Suggestions" [guidelines](https://github.com/community/community#making-suggestions) in the repo readme including: "Once you kick off a discussion, the GitHub product team will evaluate the feedback but will not be able to respond to every submission."

Thank you for contributing and continuing to support other users in the GitHub Community Discussions.
